changelog/81.feature.rst

```diff
@@ -0,0 +1,2 @@
+Create new function to calculate the temperature and emission measure from the GOES XRS measurments.
+This also removes the older verison that no longer works for the re-processed netcdf files and new GOES-R data.
```

Comment:
Can this last line be added into a breaking changelog please? 

Can you also add the before and new function names?